operator/manifests/crds/core.observatorium.io_observatoria.yaml

```diff
@@ -179,6 +179,9 @@ spec:
             compact:
               description: Thanos CompactSpec
               properties:
+                enableDownSampling:
```

Comment:
nit, if we are treating `downsampling` as one single word, then the casing should be:
```suggestion
                enableDownsampling:
```

Reply:
sorry for missing the previous comment to use `downsampling` as a single word.